patterns/1-initial/walk-the-innersource-talk.md

```diff
@@ -84,7 +88,7 @@ solve the problem:
 1. Use version-controlled repositories for all InnerSource-related assets (policies, guidelines, tooling code,
 documentation), accessible to all relevant developers.
 1. Make decisions in the open by using issue trackers and discussions rather than closed-door meetings or emails.
-1. Apply contribution workflows to internal platforms and tools exactly as recommended for InnerSource projects: open
+2. Apply contribution workflows to internal platforms and tools exactly as recommended for InnerSource projects: open
```

Comment:
Is this intentional? I think the idea was to use markdown's auto-increment here.

Reply:
Will fix